portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,19 +33,41 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private List<String> messages;
-
-  @Override
-  public void init() {
-    messages = new ArrayList<>(Arrays.asList("Hi there!", "Welcome to my protfolio", "Have a great day :)"));
-  }
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Send the JSON as the response
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
If you do not use this param, it can be a one-liner of:
DatastoreServiceFactory.getDatastoreService().put(commentEntity);

Alternatively, as It's a service (and you use it in doPost as well), the better practice is to instantiate it once (in the init()), and it may serve as a class field.